lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -910,6 +913,39 @@ export class UserAgentApplication {
     return adalFrame;
   }
 
+  private pollWindowForHash(contentWindow: Window): Promise<string> {
+      return new Promise((resolve, reject) => {
+          const intervalTimeMs = 50;
```

Comment:
Is this the right place for this? is it configurable?

Reply:
If we move this function to its own file it would probably be defined as a const outside the function, and no, wasn't planning on making it configurable, as that would be leaking implementation details.